apps/meteor/app/api/server/v1/email-inbox.ts

```diff
@@ -3,13 +3,12 @@ import { EmailInbox } from '@rocket.chat/models';
 
 import { API } from '../api';
 import { insertOneEmailInbox, findEmailInboxes, findOneEmailInbox, updateEmailInbox } from '../lib/emailInbox';
-import { hasPermission } from '../../../authorization/server/functions/hasPermission';
 import Users from '../../../models/server/models/Users';
 import { sendTestEmailToInbox } from '../../../../server/features/EmailInbox/EmailInbox_Outgoing';
 
 API.v1.addRoute(
 	'email-inbox.list',
-	{ authRequired: true },
+	{ authRequired: true, permissionsRequired: ['manage-email-inbox'] },
```

Comment:
Since we now have a permission check here, we can stop checking it again in `findEmailInboxes`